src/main/reactjs/src/utils/index.ts

```diff
@@ -33,7 +33,13 @@ export class Utils {
     return `${date}-${random}`;
   }
 
-  static formatDate(date?: Date) {
+  static optionalStringToDate(dateString?: string) {
+    if (dateString) {
+      return new Date(dateString);
+    }
+  }
+
+  static formatOptionalDate(date?: Date) {
     if (!date) {
       return '-';
     }
```

Comment:
Mulla oli tarkoituksena OPHKAKTKEH-130 yhteydessä tarkoitus siirrellä näitä dateihin liittyviä helpereitä omaan utiliteettiluokkaansa. Periaatteessa voisi jo tämän yhteydessä laittaa `utils/date.ts` alle `DateUtils`-luokan, jos haluaa. Ei pakko, saa jäädä tähänkin vielä.